webapp-django/crashstats/crashstats/tests/test_sentry.py

```diff
@@ -0,0 +1,323 @@
+"""Tests for Sentry event processing."""
+
+import pytest
+
+from ..sentry import (SanitizeBreadcrumbs,
```

Comment:
I've been avoiding relative imports like the plague because it makes it hard to move things around and know from where they actually came in the first place.

We do use them in the signature generation code so that code can exist in multiple contexts. But other than that, I'd like to avoid them.